apps/files/models.py

```diff
@@ -597,7 +597,7 @@ def save(self, *args, **kw):
         super(FileUpload, self).save()
 
     def add_file(self, chunks, filename, size):
-        filename = smart_str(filename)
+        filename = smart_str('{0}_{1}'.format(self.pk, filename))
```

Comment:
Yeah, I don't believe we're using the filename on disk in a hardcoded way anywhere. So changing it shouldn't cause any issues, and it'll only be for the newly uploaded files.